content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,106 @@
 
-## Loading Variables from Files
-
-If a workspace is configured to use Terraform 0.10.0 or later, you can commit any number of [`*.auto.tfvars` files](/docs/language/values/variables.html#variable-files) to provide default variable values. Terraform will automatically load variables from those files.
-
-If any automatically loaded variables have the same names as variables specified in the Terraform Cloud workspace, the workspace's values will override the automatic values.
-
-You can also use the optional [Terraform Cloud Provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/variable) to update a workspace's variables. This has the same effect as managing workspace variables manually or via the API, but can be more convenient for large numbers of complex variables.
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-## Variable Limits
 
-The following limits apply to variables:
 
-Component   |  Limit
-------------|---------------------------
-description |  512 characters
-key         |  128 characters
-value       |  256 kilobytes
+## Types
 
-## Managing Variables in the UI
+You can create both Terraform variables and environment variables in Terraform Cloud.
 
-To view and manage a workspace's variables, navigate to that workspace and click the "Variables" navigation link at the top.
+> **Hands-on:** The [Create Infrastructure tutorial](https://learn.hashicorp.com/tutorials/terraform/cloud-workspace-configure?in=terraform/cloud-get-started) on HashiCorp Learn shows how to set both Terraform variables and provider credential environment variables.
 
-The variables page has separate lists of Terraform variables and environment variables:
+### Terraform Variables
 
-![Screenshot: The initial appearance of a workspace's variables page](./images/vars.png)
+Terraform variables refer to [input variables](/docs/language/values/variables.html) that define parameters without hardcoding them into the configuration. For example, you could create variables that let users specify the number and type of Amazon Web Services EC2 instances they want to provision with a Terraform module. If a required input variable is missing, Terraform plans in the workspace will fail and print an explanation in the log.
 
-To edit a variable, click one of its text fields or its pencil (edit) icon to reveal the editing controls. Make any desired changes to the variable's name, value, description, and settings, then click the "Save Variable" button.
+Terraform Cloud passes input variables to Terraform by writing a `terraform.tfvars` file, meaning that Terraform Cloud overwrites any `terraform.tfvars` file you check into version control. You should not check in `terraform.tfvars` files even when running Terraform solely on the command line.
```

Comment:
The other thing I'm wondering about is whether or not readers know enough about TFC at this point to understand the statement "Terraform Cloud passes input variables to Terraform". The environment variables section introduces some of the architecture that might help users understand this statement better, so it may be better to explain that first?

Reply:
Yeah you know what - this was very confusing. I clarified with engineering and am pushing up a fix that I think makes this clearer. That's also an interesting point about knowing about TFC and how it works. I switched the order (environment variables) now come first in this section and I also modified the way it's written to hopefully make it a bit clearer as well.